runtime/tools/spec-builder/src/balances-module/index.js

```diff
@@ -0,0 +1,42 @@
+const { } = require('../wasm-loader');
+const { __retain, __getUint8Array, __allocArray, __release, getAccountDataBytes, UInt8Array_ID  } = require('../wasm-loader');
+const { Compact } = require('@polkadot/types');
+const { u8aToHex, stringToHex } = require('@polkadot/util');
+
+class Balances{
+    balances = [];
+    constructor(balances){
+        this.balances = balances;
+    }
+
+    /**
+     * 
+     * @param balances array with accountId and balances
+     */
+    static toRaw(balancesObj){
+        const rawBalances = balancesObj.balances.map(data => {
+            const key = stringToHex(data[0]);
+            const value = accDataToHex(data[1].toString());
+            return [key, value]
```

Comment:
I think that returning an object with key-value will be more readable in the `genesis` config rather than returning array, because when you are reading the variable from the `GenesisBuilder` you will do:
`result.key` or `result.value` instead of `result[0]` and `result[1]`